app/src/main/java/net/kwatts/powtools/model/OWDevice.java

```diff
@@ -105,6 +103,7 @@
 
     public final ObservableField<Boolean> isOneWheelPlus = new ObservableField<>();
 
+    public final ObservableInt mSpeedRpm = new ObservableInt();
```

Comment:
I think we shouldn't use hungarian notation, especially for this one since all the others here don't use it. ie, speedRpm instead of mSpeedRpm